apps/settings/sub_menu/exam_mode_controller.cpp

```diff
@@ -27,10 +27,10 @@ void ExamModeController::didEnterResponderChain(Responder * previousFirstRespond
 bool ExamModeController::handleEvent(Ion::Events::Event event) {
   I18n::Message childLabel = m_messageTreeModel->children(selectedRow())->label();
   if (event == Ion::Events::OK || event == Ion::Events::EXE || event == Ion::Events::Right) {
+    if (GlobalPreferences::sharedGlobalPreferences()->examMode() == GlobalPreferences::ExamMode::Activate) {
```

Comment:
Because of https://github.com/numworks/epsilon/pull/1239

```suggestion
    if (GlobalPreferences::sharedGlobalPreferences()->examMode()) {
```